src/project_types/sequence.rs

```diff
@@ -25,42 +30,180 @@ impl Sequence {
         num_sections: Option<u32>
     ) -> Result<Sequence, Error> {
         // Defaults
+        // TODO: put constraint (>25ms)
         let frame_dur_ms = frame_duration_ms.unwrap_or(50);
+        // TODO: put constraint (!= 0)
```

Comment:
Will do, but also need to add these two variables to new-sequence command. Opening new issue for that